test/AspNetCoreModuleTests/NativeTests.targets

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <PropertyGroup>
+    <VsTestConsole>$(VCIDEInstallDir)..\CommonExtensions\Microsoft\TestWindow\vstest.console.exe</VsTestConsole>
```

Comment:
Did you try using `dotnet vstest`?

Reply:
Dotnet fails on imports in vcprojes.

Reply:
Played with it a little more just b/c I wanted to understand what happens here. It looks like there is a test adapter that ships only in VS. vstest.console itself in `dotnet vstest` is capable of handling this, but the adapter is not shipped in the CLI sdk.